perf/jmh/run_jmh_jdk_micros.sh

```diff
@@ -0,0 +1,239 @@
+#                              are running. (default: true)
+synchronize_iterations_opt="-si true"
+
+#  -i <int>                    Number of measurement iterations to do. Measurement 
+#                              iterations are counted towards the benchmark score. 
+#                              (default: 1 for SingleShotTime, and 5 for all other 
+#                              modes)
+measurement_iterations_opt="-i $measurement_iterations"
+
+#  -w <time>                   Minimum time to spend at each warmup iteration. Benchmarks
+#                              may generally run longer than iteration duration.
+#                              (default: 10 s)
+warmup_time_opt="-w $warmup_time"
+
+#  -wi <int>                   Number of warmup iterations to do. Warmup iterations 
+#                              are not counted towards the benchmark score. (default: 
+#                              0 for SingleShotTime, and 5 for all other modes)
+warmup_iterations_opt="-wi $warmup_iterations"
+
+#  -t <int>                    Number of worker threads to run with. 'max' means 
+#                              the maximum number of hardware threads available 
+#                              on the machine, figured out by JMH itself. (default: 
+#                              1)
+worker_threads_opt="-t $num_worker_threads"
+
+#  -o <filename>               Redirect human-readable output to a given file.
+human_readable_output_file_opt="-o $human_readable_output_file"
+
+#  -rff <filename>             Write machine-readable results to a given file. 
+#                              The file format is controlled by -rf option. Please 
+#                              see the list of result formats for available formats. 
+#                              (default: jmh-result.<result-format>)
+machine_readable_output_file_opt="-rff $machine_readable_output_file"
+
+#  -rf <type>                  Format type for machine-readable results. These 
+#                              results are written to a separate file (see -rff). 
+#                              See the list of available result formats with -lrf. 
+#                              (default: CSV)
+#
+# Output from `java $benchmark_jar_path -lrf`:
+#  Available formats: text, csv, scsv, json, latex
+format_type_opt="-rf text"
+
+benchmark="-jar $benchmark_jar_path"
+benchmark="$benchmark $fork_opt $synchronize_iterations_opt $warmup_time_opt"
+benchmark="$benchmark $measurement_iterations_opt $warmup_iterations_opt"
+benchmark="$benchmark $worker_threads_opt $human_readable_output_file_opt"
+benchmark="$benchmark $machine_readable_output_file_opt $format_type_opt"
+
+jdk_bin_path="${TEST_JDK_HOME}/bin/"
+echo "Info : JDK home is : $jdk_bin_path "
+
+if [ ! -d "$jdk_bin_path" ]; then
+    echo "Warning: Could not find JDK bin path: $jdk_bin_path"
+    echo "Warning: Using the java binary automatically selected by the shell."
+    jdk_bin_path=""
+fi
+
+run_benchmarks()
```

Comment:
Perhaps move this function to the top together with the other ones.

Reply:
@JohnTortugo The necessary variables required to execute the function are set prior to its execution. Hence, it's uncertain whether we can relocate the function to the beginning of the script.